miniml/compiler/compile.scm

```diff
@@ -1793,6 +1823,98 @@
               ))
     )))
 
+(define (compile-letexits env stacksize exit-defs body)
+  (let* (
+      (endlab (newlabel))
+      (exit-labels (map (lambda (def) (newlabel)) exit-defs))
+      (exit-names (map car exit-defs))
+      (exit-varss (map cadr exit-defs))
+      (exit-bodies (map caddr exit-defs))
+      (exit-infos
+       (map (lambda (exit-label exit-vars)
+               (compute-exit-info env stacksize exit-label exit-vars)
+         ) exit-labels exit-varss))
+      (body-env
+       (fold (lambda (name info env) (compenv-add-exit env name info))
+           env exit-names exit-infos))
+      (max-arity
+       (fold (lambda (info max-arity) (max (exit-info-get-arity info) max-arity))
+           0 exit-infos))
+    )
+    ; Each exit needs an "argument space" of size exit-arity reserved on the stack.
+    ; Jumping to one of the exits leaves the scope were those exits are available,
+    ; so we will jump to at most one of these exit.
+    ; We reserve overlapping space for their arguments, of size max-arity.
+    (bytecode-put-u32-le ATOM0)
```

Comment:
Why is `Atom(0)` used as a filler here, instead of something like the integer `0`?

Reply:
ATOM0 fits in a single opcode, while constant integers take two. Do you prefer constant integers?

Reply:
I have a slight preference for constant integers, but `Atom(0)` is fine too, so just leave it as is. However, there are the `CONSTn` instruction for n=0,1,2,3 which exist as well.

Reply:
I rebased to use CONST0 instead, thanks!